scalafix-tests/input/src/main/scala/test/DisableUnlessInsideBlock.scala

```diff
@@ -1,11 +1,11 @@
 /*
 rules = [
-  DisableUnless
+  Disable
 ]
 
-DisableUnless.symbols = [
+Disable.unlessInsideBlock = [
   {
-    unless = "test.DisableUnless.IO"
+    unless = "test.DisableUnlessInsideBlock.IO"
```

Comment:
Should this field be called "safeBlock" or "block" instead of "unless"?

```
Disable.unlessInsideBlock = [
  { safeBlock = "IO",
    disabledSymbols = ... }
]
```
?

Any thoughts @fommil ? I personally haven't used this rule, but "unless = X, symbols = YY" seems weird.

Reply:
the names of these things are starting to get quite big, but I think a lot of symbols will have a shared `safeBlock` so it makes sense for them to be grouped like this. Each symbol may have a different explanation, though, so as long as that is supported we're good.